__tests__/views/Home.spec.js

```diff
@@ -8,7 +8,7 @@ import emails from "../../src/store/modules/emails";
 jest.mock("../../src/store/modules/emails");
 
 describe("Test Home.vue view", () => {
-  it("Check for h1 exists", () => {
+  it("Check for main exists", () => {
```

Comment:
We can still check to see if h1 is being rendered by adding these getters to the mocked inbox module and adding the mocked inbox to the test state

const getters = {
  getSelectedInboxItem() {
    return 1;
  },
  getInboxItems(state, getters, rootState, rootGetters) {
    return [];
  },
  isLoaded(state, getters, rootState) {
    return true;
  },
};

Reply:
Thank you! :)